R/correctionsataglance-data.R

```diff
@@ -203,7 +203,11 @@ parseCorrThresholds <- function(reportObject, timezone){
       type <- rep(threshold[['type']], times=nrow(periods))
       startDates <- periods[['startTime']]
       endDates <- periods[['endTime']]
-      value <- periods[['referenceValue']]
+      if(isEmptyOrBlank(periods[['referenceValue']])){
+        value <- NA
+      } else {
+        value <- periods[['referenceValue']]
+      }
```

Comment:
@dpattermann-usgs yep! Just not normal R syntax to do it in one line, and I don't know if that has implications down the road.